scripts/build-android.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+NODE_VERSION=`node -v`
+if [ $NODE_VERSION != "v11.6.0" ]
+then
+  echo "[ERROR] Node version needs to be v11.6.0."
+  exit
```

Comment:
Can we exit with an error code like `exit 1`?

Otherwise other tooling might consider this failure to be a successful build.